docs/user/integration/dynatrace/README.md

```diff
@@ -82,22 +84,21 @@ As a result, you see data arriving in your environment, advanced Kubernetes moni
 
 Next, you set up the ingestion of custom span and Istio span data, and, optionally, custom metrics based on OTLP.
 
-### Create Access Token
+### Create Access Token and Secret
+>**NOTE:** if you have already created `dynakube` Secret in prerequisites then skip this step.
+1. To push custom metrics and spans to Dynatrace, set up a [dataIngestToken](https://docs.dynatrace.com/docs/manage/access-control/access-tokens).
 
-To push custom metrics and spans to Dynatrace, set up an [API Token](https://docs.dynatrace.com/docs/manage/access-control/access-tokens).
+   Follow the instructions in [Dynatrace: Generate an access token](https://docs.dynatrace.com/docs/manage/access-control/access-tokens#create-api-token) and select the following scopes:
```

Comment:
are those scopes or templates (like in line 96)?

Reply:
both :), template is bunch of scopes and are pre-defined in dynatrace. There is no template defined for dataIngestToken so we have to mention specific scopes.